### PR TITLE
rename token to apikey

### PR DESCRIPTION
**Pull request by mthenw**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -94,15 +94,17 @@ const SDK = require('@serverless/event-gateway-sdk');
 const eventGateway = new SDK({
   url: 'http://localhost',
```
Comment (alexdebrie) [inline README.md]: Should we change this to `eventsUrl` while we're at it, or is `url` clear enough?

File: lib/index.js [hunk 2]
```diff
@@ -23,13 +23,13 @@ class EventGateway {
     }
 
     const config = {
-      apiUrl: configuration.url,
+      eventsUrl: configuration.url,
```
Comment (alexdebrie) [inline lib/index.js]: Same Q as above -- would need to change this to `configuration.eventsUrl`

File: README.md (hunk 1 above)
Comment (mthenw) [reply README.md]: yup, makes sense. updating...

Comment (mthenw) [reply README.md]: on the other hand this `url` looks cool. Let's leave as it is for now.

**Pull request merged**